inbm/dispatcher-agent/dispatcher/source/linux_gpg_key.py

```diff
@@ -21,7 +22,7 @@ def remove_gpg_key_if_exists(gpg_key_name: str) -> None:
     try:
         key_path = os.path.join(LINUX_GPG_KEY_PATH, gpg_key_name)
         if os.path.exists(key_path):
-            os.remove(key_path)
+            remove_file(key_path)
```

Comment:
The change to use `remove_file` from `inbm_common_lib.utility` is good for consistency, but ensure that the `remove_file` function handles all necessary exceptions and logging. [medium]